spec/models/manageiq/providers/amazon/network_manager/stubbed_refresher_spec.rb

```diff
@@ -0,0 +1,180 @@
+require_relative '../aws_helper'
+require_relative '../aws_stubs'
+
+describe ManageIQ::Providers::Amazon::NetworkManager::Refresher do
+  include AwsStubs
+
+  describe "refresh" do
+    before(:each) do
+      _guid, _server, zone = EvmSpecHelper.create_guid_miq_server_zone
+      @ems = FactoryGirl.create(:ems_amazon, :zone => zone)
+      @ems.update_authentication(:default => {:userid => "0123456789", :password => "ABCDEFGHIJKL345678efghijklmno"})
+    end
+
+    let(:ec2_user) { FactoryGirl.build(:authentication).userid }
+    let(:ec2_pass) { FactoryGirl.build(:authentication).password }
+    let(:ec2_user_other) { 'user_other' }
+    let(:ec2_pass_other) { 'pass_other' }
+    subject { described_class.discover(ec2_user, ec2_pass) }
+
+    before do
+      EvmSpecHelper.local_miq_server(:zone => Zone.seed)
+
+      stub_load_balancer_service
+    end
+
+    around do |example|
+      with_aws_stubbed(:ec2 => stub_responses) do
```

Comment:
If you stub the `elasticloadbalancer` service, you'd have to add that as a key to the options hash here